src/storybook-snapper.tsx

```diff
@@ -126,13 +140,18 @@ export function story(storyName, cb) {
 function runSnap(
   snapshotName: string,
   cb: ChildrenProp,
-  ignore: boolean = false,
+  runBefore?: RunBefore,
+  ignore?: boolean,
 ) {
   const eyesStorybookOptions: EyesStorybookOptions = {
-    waitBeforeScreenshot: `[${DATA_READY_HOOK}="true"]`,
+    runBefore,
   };
   const fullStoryName = [...currentTest].join('/');
 
+  if (typeof cb === 'function' && functionUsesArgs(cb)) {
```

Comment:
consider to include inside `functionUsesArgs` also the `cb===function`, it's repeated above.
also, rename to something more meaningful like "isStoryInteractive"